timeless/restaurants/models.py

```diff
@@ -4,17 +4,19 @@
 from timeless.db import DB
 from timeless.models import TimestampsMixin
 from timeless.poster.models import PosterSyncMixin
+from sqlalchemy_utils import ChoiceType
 
 
-class ReservationStatus(enum.Enum):
-    """Reservation status"""
-    unconfirmed = 1
-    confirmed = 2
-    started = 3
-    finished = 4
-    canceled = 5
-    late = 6
-    not_contacting = 7
+"""Reservation status"""
+RESERVATION_STATUS = [
+    (u"unconfirmed", u"unconfirmed"),
+    (u"confirmed", u"confirmed"),
+    (u"started", u"started"),
+    (u"finished", "finished"),
+    (u"canceled", u"canceled"),
+    (u"late", u"late"),
+    (u"not_contacting", u"not_contacting")
```

Comment:
@emilianodellacasa this seems strange -- identical key/value pairs?

Reply:
@amihaiemil Will fix it